src/stories/Jobs/CanCloseJob/endpoint.spec.js

```diff
@@ -0,0 +1,43 @@
+const contextClassRef = requireUtil("contextHelper");
+const randomUser = requireUtil("randomUser");
+const knex = requireKnex();
+const httpServer = requireHttpServer();
+const JobsRepo = requireRepo("jobs");
+
+describe("Test API CanCloseJob", () => {
+  beforeAll(async () => {
+    contextClassRef.user = randomUser();
+    contextClassRef.headers = {
+      Authorization: `Bearer ${contextClassRef.user.token}`, // An authenticated user is making the api call
+    };
+  });
+
+  it("can_close_job", async () => {
+    let respondResult;
+    try {
+      const app = httpServer();
+
+      const testJob = await JobsRepo.create({
+        recruiter_uuid: "001",
```

Comment:
let's also test it with a random uuid?

Reply:
i tested it was working will add same here